packages/react/src/lib/whatInput.ts

```diff
@@ -145,37 +134,14 @@ const setInput = event => {
         window.sessionStorage.setItem('what-input', currentInput)
       } catch (e) {}
 
-      doUpdate()
+      doUpdate(event.view.document)
     }
   }
 }
 
 // updates the doc and `inputTypes` array with new input
-const doUpdate = () => {
-  docElem.setAttribute(`data-whatinput`, currentInput)
-}
-
-const setElement = event => {
-  if (!event.target.nodeName) {
-    // If nodeName is undefined, clear the element
-    // This can happen if click inside an <svg> element.
-    clearElement()
-    return
-  }
-
-  currentElement = event.target.nodeName.toLowerCase()
-  docElem.setAttribute('data-whatelement', currentElement)
-
-  if (event.target.classList && event.target.classList.length) {
-    docElem.setAttribute('data-whatclasses', event.target.classList.toString().replace(' ', ','))
-  }
-}
```

Comment:
We don't use `data-whatclasses` & `data-whatelement`, so it's safe to remove them